sklearnex/tests/test_memory_usage.py

```diff
@@ -114,7 +114,7 @@ def gen_functions(functions):
 
 data_shapes = [
     pytest.param((1000, 100), id="(1000, 100)"),
-    pytest.param((2000, 50), id="(2000, 50)"),
+    pytest.param((2000, 40), id="(2000, 40)"),
```

Comment:
This change alone makes memory usage test failures. @icfaust, what is the reason for it?

Reply:
Test failures disappear locally if data shape change is reverted. Not sure if it will be the same in CI.
Lower data shapes are more likely to cause failures because base memory usage takes larger percentage of data size.